conans/tools.py

```diff
@@ -64,7 +64,7 @@ def unzip(filename, destination="."):
                 # the limit is 230 to account for 10 chars of SHA in b/xxxx (in shorted paths)
                 # and 20 chars for build system extra subfolders (cmake mytarget.dir/debug, etc)
                 # Win limit is 260
-                if len(file_.filename) + len(full_path) > 230:
+                if len(file_.filename) + len(full_path) >= 260:
```

Comment:
This unzip is now able to unzip larger paths, but then it warns in the copy from source => build, that also has a check for path length. I have tested to build Boost in Win (unzipping boost had this issue) and works fine. Maybe the copy source=>build is slow, but nothing really changed there, it was already slow I guess.